src/scss/partial/_grid.scss

```diff
@@ -5,7 +5,7 @@ $menu-sub-nav-width: 200px;
 $columns: 12;
 
 $grid-gap: 24px;  // gap for normal layout
-$grid-gap-dev: 20px;  // gap for dev/admin layout
+$grid-gap-dev: 24px;  // gap for dev/admin layout
```

Comment:
gap for dev/admin layout is `20px`, no need to change

Reply:
ok